### ISSUE TITLE
Floris board temprarily crashed in firefox lite browser on android 9

### ISSUE THREAD

**Issue by nascentchlorine** [florisboard/florisboard#566]
I am new to GitHub but here is the log.

~~~ 1617466877417.stacktrace ~~~

java.lang.IndexOutOfBoundsException: Empty list doesn't contain element at index -1.
	at kotlin.collections.EmptyList.get(Collections.kt:36)
	at kotlin.collections.EmptyList.get(Collections.kt:24)
	at dev.patrickgold.florisboard.ime.core.EditorInstance.deleteWordsBeforeCursor(EditorInstance.kt:235)
	at dev.patrickgold.florisboard.ime.text.TextInputManager.handleDeleteWord(TextInputManager.kt:468)
	at dev.patrickgold.florisboard.ime.text.TextInputManager.onInputKeyUp(TextInputManager.kt:704)
	at dev.patrickgold.florisboard.ime.core.InputEventDispatcher$1$3.invokeSuspend(InputEventDispatcher.kt:125)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at android.os.Handler.handleCallback(Handler.java:873)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:201)
	at android.app.ActivityThread.main(ActivityThread.java:6810)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:547)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:873)

**Comment by Glitchy-Tozier** [comment]
Hi!  :)

What Version of the app were you using and do you remember wat you did on your keyboard that made this happen?

(Also a small heads-up, this might be a duplicate of bug #484. Let's see what Patrick thinks.)

**Comment by nascentchlorine** [comment]
> Hi! :)
> 
> What Version of the app were you using and do you remember wat you did on your keyboard that made this happen?

Yeah it seems it like it is that bug. It showed the error message :"Keyboard has stopped working." I am using version 0.3.9 from F-droid. But the thing is it was only once and now it works just fine. As I correctly remember I was logging in into a site and copied a password onto the clipboard. But after I pasted it the problem showed up.

**Comment by patrickgold** [comment]
Yes this is a stacktrace I've seen too often in the last few days. In v0.3.9 it is definitely in, but the next stable release v0.3.10 isn't too far away (next weekend), so hopefully this crash will not occur anymore. Closing this issue as a duplicate.

**Comment by nascentchlorine** [comment]
> Yes this is a stacktrace I've seen too often in the last few days. In v0.3.9 it is definitely in, but the next stable release v0.3.10 isn't too far away (next weekend), so hopefully this crash will not occur anymore. Closing this issue as a duplicate.
 Great!

**Comment by nascentchlorine** [comment]
> Yes this is a stacktrace I've seen too often in the last few days. In v0.3.9 it is definitely in, but the next stable release v0.3.10 isn't too far away (next weekend), so hopefully this crash will not occur anymore. Closing this issue as a duplicate.

Great!
